source/dpp/runtime/app.d

```diff
@@ -243,7 +243,7 @@ private void runCPreProcessor(in string cppPath, in string tmpFileName, in strin
     const cppArgs = [cpp, tmpFileName];
     const ret = () {
         try
-            return execute(cppArgs);
+            return execute(cppArgs, null, Config.stderrPassThrough);
```

Comment:
The `null` here is confusing to the reader. In these cases, please either add a `/* env */` comment right after the argument, or name a variable so that the reader knows what this `null` refers to.